tools/buildsys/src/gomod/error.rs

```diff
@@ -0,0 +1,42 @@
+use std::path::PathBuf;
+
+use snafu::Snafu;
+
+#[derive(Debug, Snafu)]
+#[snafu(visibility(pub(super)))]
+pub(crate) enum Error {
+    #[snafu(display("Failed to start command: {}", source))]
+    CommandStart { source: std::io::Error },
+
+    #[snafu(display("Failed to execute command: 'docker {:?}'", args))]
+    DockerExecution { args: String },
+
+    #[snafu(display("input url is required"))]
+    InputFile,
+
+    #[snafu(display("input file {:?} must be a file", path))]
+    InputFileBad { path: PathBuf },
+
+    #[snafu(display("Bad file url '{}': {}", url, source))]
+    InputUrl {
+        url: String,
+        source: url::ParseError,
+    },
+
+    #[snafu(display("mod-dir is required"))]
+    ModDir,
+
+    #[snafu(display("output-dir is required"))]
+    OutputDir,
+
+    #[snafu(display("output_dir must not exist or be a directory"))]
+    OutputDirBad,
```

Comment:
This error isn't used, you could remove it.

Reply:
Good catch - thanks!